src/Components/MailtoDropdown.tsx

```diff
@@ -0,0 +1,171 @@
+        countries: ['all'],
+    },
+    yandex: {
+        link: (d) => `https://mail.yandex.com/#compose?to=${d.email}&subject=${d.subject}&body=${d.body}`,
+        countries: ['all'],
+    },
+    copymanually: {
+        onClick: (d, showCopyManuallyModal) => showCopyManuallyModal(),
+        countries: ['all'],
+    },
+});
+
+export const MailtoDropdown = (props: MailtoDropdownProps) => {
+    const my_ref_text = `${t_r('my-reference', props.letter.props.language)}: ${props.letter.props.reference}`;
+    const data = {
+        email: props.email,
+        subject: encodeURIComponent(
+            props.letter.props.subject ? `${props.letter.props.subject} (${my_ref_text})` : my_ref_text
+        ),
+        body: encodeURIComponent(props.letter.toEmailString()),
+    };
+
+    const onCopyManuallyInputClick = (e: JSX.TargetedMouseEvent<HTMLInputElement | HTMLTextAreaElement>) => {
+        if (previous_active_element.id === e.currentTarget.id) return;
+
+        e.currentTarget.select();
+        e.currentTarget.focus();
+        previous_active_element = e.currentTarget;
+    };
+    const [CopyManuallyModal, showCopyManuallyModal, dismissCopyManuallyModal] = useModal(
+        <IntlProvider scope="generator" definition={window.I18N_DEFINITION}>
+            <Text id="copymanually-explanation" />
+
+            <div className="form-group">
+                <strong>
+                    <label htmlFor="mailto-dropdown-copymanually-subject">{t('subject', 'generator')}</label>
+                </strong>
+                <input
+                    type="text"
+                    id="mailto-dropdown-copymanually-subject"
+                    className="form-element"
+                    value={decodeURIComponent(data.subject)}
+                    onClick={onCopyManuallyInputClick}
+                    readOnly
+                />
+                <strong>
+                    <label htmlFor="mailto-dropdown-copymanually-recipient">{t('recipient', 'generator')}</label>
+                </strong>
+                <input
+                    type="text"
+                    id="mailto-dropdown-copymanually-recipient"
+                    className="form-element"
+                    value={data.email}
+                    onClick={onCopyManuallyInputClick}
+                    readOnly
+                />
+                <strong>
+                    <label htmlFor="mailto-dropdown-copymanually-body">{t('body', 'generator')}</label>
+                </strong>
+                <textarea
+                    id="mailto-dropdown-copymanually-body"
+                    className="form-element"
+                    rows={10}
+                    onClick={onCopyManuallyInputClick}
+                    readOnly>
+                    {decodeURIComponent(data.body)}
+                </textarea>
+            </div>
+        </IntlProvider>,
+        { onPositiveFeedback: () => dismissCopyManuallyModal(), positiveText: t('ok', 'generator') }
```

Comment:
`previous_active_element` should be reset on dismiss!

Reply:
I know my fix doesn't solve the problem when you just click outside the modal. But _please_ don't make me bring back `onDismissed` just for that. .D